certbot/cli.py

```diff
@@ -730,14 +765,14 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         help="(certbot-auto only) prevent the certbot-auto script from"
              " upgrading itself to newer released versions")
     helpful.add(
-        "automation", "-q", "--quiet", dest="quiet", action="store_true",
+        ["automation", "renew"],
```

Comment:
Maybe `certonly`. I care less about this one, but there are a number of people still using `certonly` for renewal. We should try to stop people from doing that, but in the meantime, perhaps it should be listed here?